### PR TITLE
Updating data augmentation tutorial to include Keras preprocessing utilities

### PR DESCRIPTION
**Pull request by mjang2000**

### CODE REVIEW

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 1]
```diff
@@ -377,44 +486,98 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "HKqeahG2hVdV"
+      },
+      "source": [
+        "With this approach, you use `Dataset.map` to create a dataset that yields batches of augmented images. In this case:\n",
+        "\n",
+        "* Data augmentation will happen asynchronously on the CPU, and is non-blocking. You can overlap the training of your model on the GPU with data preprocessing, using `Dataset.prefetch`, shown below.\n",
+        "* In this case, you will need to reimplement preprocessing logic server-side, as the prepreprocessing layers will not be exported with the model when you call `model.save`.\n"
```
Comment (MarkDaoust) [inline site/en/tutorials/images/data_augmentation.ipynb]: Yes, but I would also mention that after training you can attach the preprocessing before export.

**Comment by MarkDaoust** [comment]
I'm happy with this. @lamberta any feedback before I merge?

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 2]
```diff
@@ -110,49 +108,57 @@
...
       "metadata": {
         "colab_type": "text",
-        "id": "__EuXwM-8uth"
+        "id": "Ydx3SSoF4wpG"
       },
       "source": [
-        "Let's check the data augmentation features on an image and then augment a whole dataset later to train a model."
+        "## Download a dataset\n",
+        "\n",
+        "In this tutorial you will use the [tf_flowers](https://www.tensorflow.org/datasets/catalog/tf_flowers) dataset. For convenience, we will download this dataset using [TensorFlow Datasets](https://www.tensorflow.org/datasets). If you would like to learn about others ways of importing data, see the [load images](https://www.tensorflow.org/tutorials/load_data/images) tutorial.\n"
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:133]:
```suggestion
        "This tutorial uses the [tf_flowers](https://www.tensorflow.org/datasets/catalog/tf_flowers) dataset. For convenience, download the dataset using [TensorFlow Datasets](https://www.tensorflow.org/datasets). If you would like to learn about others ways of importing data, see the [load images](https://www.tensorflow.org/tutorials/load_data/images) tutorial.\n"
```

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 3]
```diff
@@ -185,22 +191,55 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vdJ6XA4q2nqK"
+      },
+      "source": [
+        "## Using Keras Preprocessing Layers"
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:212]:
```suggestion
        "## Use Keras preprocessing layers"
```

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 4]
```diff
@@ -327,23 +383,48 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MxGvUT727Po6"
+      },
+      "source": [
+        "**Option 1: Make the preprocessing layers part of your model**"
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:427]:
```suggestion
        "#### Option 1: Make the preprocessing layers part of your model"
```

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 5]
```diff
@@ -352,23 +433,51 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "B2X3JTeY_vfv"
+      },
+      "source": [
+        "**Option 2: Apply the preprocessing layers to your dataset**"
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:480]:
```suggestion
        "#### Option 2: Apply the preprocessing layers to your dataset"
```

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 6]
```diff
@@ -377,44 +486,98 @@
...
     {
       "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
-        "id": "8W5E_c7o-H96"
+        "id": "cgj51k9J7jfc"
       },
       "source": [
-        "See the `tf.image` reference for details about available augmentation options."
+        "You can find an example of the first option in the [Image Classification](https://www.tensorflow.org/tutorials/images/classification) tutorial. We will demonstrate the second option here."
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:517]:
```suggestion
        "You can find an example of the first option in the [image classification](https://www.tensorflow.org/tutorials/images/classification) tutorial. We will demonstrate the second option here."
```

Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:517]:
```suggestion
        "You can find an example of the first option in the [Image Classification](https://www.tensorflow.org/tutorials/images/classification) tutorial. Let's demonstrate the second option here."
```

File: site/en/tutorials/images/data_augmentation.ipynb [hunk 7]
```diff
@@ -649,25 +1004,93 @@
...
         "colab_type": "text",
-        "id": "UEqeeNsHZaC5"
+        "id": "isrM-MZtpxTq"
       },
       "source": [
-        "## Conclusion:\n",
+        "### Apply augmentation to a dataset\n",
         "\n",
-        "In this example the augmented model converges to an accuracy ~95% on validation set. This is slightly higher (+1%) than the model trained without data augmentation."
+        "As before, we can apply data augmentation to a dataset using `Dataset.map`."
```
Comment (lamberta) [inline site/en/tutorials/images/data_augmentation.ipynb:1076]:
```suggestion
        "As before, apply data augmentation to a dataset using `Dataset.map`."
```

**Comment by mjang2000** [comment]
Thank you for the feedback! The edits have been made

**Pull request merged**